internal/output/output_result.go

```diff
@@ -94,22 +96,30 @@ type SeverityCount struct {
 }
 
 type VulnTypeCount struct {
-	All      int
-	OS       int
-	Project  int
-	Uncalled int
+	All     int
+	OS      int
+	Project int
+	Hidden  int
 }
 
-type CallAnalysisCount struct {
-	Called   int
-	Uncalled int
+type AnalysisCount struct {
```

Comment:
Can you add a comment here and above on VulnTypeCount to explain why there is two different counts?

Reply:
renamed `VulnTypeCount` to `VulnTypeSummary` and added comments. The `VulnTypeSummary` here is mostly used to implement the HTML vuln type filter.